installation/downloadReferences.sh

```diff
@@ -31,118 +31,128 @@ mkdir_cd hg19_GRCh37_1000genomes
+    check_md5sum && exit 0 || echo downloading mappability file....
 
-	wget -c https://raw.githubusercontent.com/eilslabs/ACEseqWorkflow/github/installation/ReplicationTime_10cellines_mean_10KB.Rda
+    wget -c http://hgdownload.soe.ucsc.edu/goldenPath/hg19/encodeDCC/wgEncodeMapability/wgEncodeCrgMapabilityAlign100mer.bigWig
+    bigWigToBedGraph wgEncodeCrgMapabilityAlign100mer.bigWig /dev/stdout | bgzip > wgEncodeCrgMapabilityAlign100mer_chr.bedGraph.gz
+    tabix -p bed wgEncodeCrgMapabilityAlign100mer_chr.bedGraph.gz
+    rm -f wgEncodeCrgMapabilityAlign100mer.bigWig
 
-	check_md5sum
+    check_md5sum
 )
 
 ###############################################################################
-# download chromosome statistics
+# download replication timings
 ###############################################################################
 (
-	mkdir_cd stats
+    mkdir_cd databases/ENCODE
 
-	EXPECTED_MD5SUM=801bdaa8c3b0d5c18a0637b0b29fd337
-	check_md5sum && exit 0 || echo downloading stats files....
+    EXPECTED_MD5SUM=2a63b34a737383af2a3f7eb32801a5fa
+    check_md5sum && exit 0 || echo downloading replication timing file....
 
-	wget -c http://hgdownload.cse.ucsc.edu/goldenPath/hg19/database/chromInfo.txt.gz
-	zcat chromInfo.txt.gz | grep -Pv "(_)|(chrM)" | sed -e '1i\#chrom\tsize\tfileName' > chrlengths.txt
-	rm -f chromInfo.txt.gz
+    wget -c https://raw.githubusercontent.com/eilslabs/ACEseqWorkflow/github/installation/ReplicationTime_10cellines_mean_10KB.Rda
 
-	wget -c https://raw.githubusercontent.com/eilslabs/ACEseqWorkflow/github/installation/hg19_GRch37_100genomes_gc_content_10kb.txt
-
-	check_md5sum
+    check_md5sum
 )
 
 ###############################################################################
-# download IMPUTE database
+# download chromosome statistics
 ###############################################################################
 (
-	mkdir_cd databases/1000genomes/IMPUTE
+    mkdir_cd stats
 
-	EXPECTED_MD5SUM=261a28d6b6917340cd82ada2d7185e17
-	check_md5sum && exit 0 || echo downloading impute files....
+    EXPECTED_MD5SUM=801bdaa8c3b0d5c18a0637b0b29fd337
+    check_md5sum && exit 0 || echo downloading stats files....
 
-	wget -c https://mathgen.stats.ox.ac.uk/impute/ALL.integrated_phase1_SHAPEIT_16-06-14.nomono.tgz
-	tar -xzvf ALL.integrated_phase1_SHAPEIT_16-06-14.nomono.tgz
-	rm -f ALL.integrated_phase1_SHAPEIT_16-06-14.nomono.tgz
+    wget -c http://hgdownload.cse.ucsc.edu/goldenPath/hg19/database/chromInfo.txt.gz
+    zcat chromInfo.txt.gz | grep -Pv "(_)|(chrM)" | sed -e '1i\#chrom\tsize\tfileName' > chrlengths.txt
+    rm -f chromInfo.txt.gz
 
-	wget -c https://mathgen.stats.ox.ac.uk/impute/ALL_1000G_phase1integrated_v3_impute.tgz
-	tar -xzvf ALL_1000G_phase1integrated_v3_impute.tgz
-	rm -f ALL_1000G_phase1integrated_v3_impute.tgz
+    wget -c https://raw.githubusercontent.com/eilslabs/ACEseqWorkflow/github/installation/hg19_GRch37_100genomes_gc_content_10kb.txt
 
-	check_md5sum
+    check_md5sum
```

Comment:
The diff was completely messed up in github. Is it possible to move the inserted beagle stuff downwards -- just where impute used to be before?